certified-connectors/DocuSignDemo/script.csx

```diff
@@ -2976,6 +2976,153 @@ public class Script : ScriptBase
     return body;
   }
 
+  private JObject BulkSendBodyTransformation(JObject body)
+  {
+    var query = HttpUtility.ParseQueryString(this.Context.Request.RequestUri.Query);
+    var name = query.Get("name");
+    JObject newBody = ParseCSV(body);
+    newBody["name"] = name;
+    return newBody;
+  }
+
+  private JObject ParseCSV(JObject inputBody)
+  {
+    var input = inputBody.GetValue("csv").ToString();
+    var body = new JObject();
+
+    var result = new JObject();
+
+    try
+    {
+        var lines = input.Split(new string[] { "\r\n", "\r", "\n" }, StringSplitOptions.None);
+
+        var headerLine = lines[0];
+        var headerItems = headerLine.Split(',');
+
+        var parsedHeaders = new string[headerItems.Length][];
+
+        for (int i = 0; i < headerItems.Length; i++)
```

Comment:
Place all declarations at the beginning of the try block. This for loop can be moved after the declarations. These small changes improve the readability of the code.

Reply:
Sure.

Reply:
Done.